lib/main.coffee

```diff
@@ -109,3 +111,27 @@ module.exports =
         console.warn('Copying Markdown as HTML failed', error)
       else
         atom.clipboard.write(html)
+
+  saveAsHtml: ->
+    activePane = atom.workspace.getActivePaneItem()
+    if isMarkdownPreviewView(activePane)
+      activePane.saveAs()
+      return
+
+    editor = atom.workspace.getActiveTextEditor()
+    return unless editor?
+
+    grammars = atom.config.get('markdown-preview.grammars') ? []
+    return unless editor.getGrammar().scopeName in grammars
+
+    uri = @uriForEditor(editor)
+    previewPane = atom.workspace.paneForURI(uri)
```

Comment:
I would rename this markdownPreviewPane to be more explicit (especially since there used to be a Tabs setting called "Preview Panes" or similar).